pkg/plugins/manager/signature/manifest.go

```diff
@@ -213,7 +213,7 @@ func Calculate(log log.Logger, plugin *plugins.Plugin) (plugins.Signature, error
 	}, nil
 }
 
-func verifyHash(pluginID string, path string, hash string) error {
+func verifyHash(log log.Logger, pluginID string, path string, hash string) error {
```

Comment:
Nit. Would name the logger something else than `log` since it overrides the log package name.